log/log.go

```diff
@@ -27,11 +27,33 @@ const (
 	ZapNopLevel zapcore.Level = zapcore.FatalLevel + 1
 )
 
-// InitLogger provides a quick way to start or replace a logger
-// Pass in debug as log level enables development mode (which makes DPanicLevel logs panic).
+// ToAtomicLevel converts Level to a zap acceptable atomic level struct
```

Comment:
What does “atomic level” mean here? It might be good to clarify that in the comment for future devs.

Reply:
Done.